marklogic-unit-test-client/src/test/ml-modules/root/test/suites/Helpers/Remove Modules/test-remove-modules.xqy

```diff
@@ -0,0 +1,17 @@
+xquery version "1.0-ml";
+
+import module namespace test="http://marklogic.com/test/unit" at "/test/test-helper.xqy";
+import module namespace td="remove-modules-module" at "test-data/td.xqy";
+
+let $_ := test:remove-modules(map:keys($td:TEST-DATA))
+
+return 
+  xdmp:invoke-function(
```

Comment:
Same trick where you can use `;` to start a different transaction.  No need to use xdmp:invoke-function to get transaction isolation.